### PR TITLE
New Template: hp_comware_display_lldp_neighbor-information_verbose

### PR DESCRIPTION
**Pull request by xdai555**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

hp_comware_display_lldp_neighbor-information_verbose,hp_comware,display lldp neighbor-information verbose
##### SUMMARY

Gather neighbor lists by lldp verbose.
Do more things for hp_comware~

### CODE REVIEW

File: templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Value Required LOCAL_IF (\S+)
```
Comment (FragmentedPacket) [inline templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm]: Can you please update the value names to match other LLDP templates (e.g. cisco_ios_show_lldp_neighbors_detail.textfsm)

The **CHASSIS_ID** may be better as `\S+` as well.

File: templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Value NEIGHBOR (\S+)
+Value CHASSIS_ID ([\d\w-]+)
+Value NEIGHBOR_PORT (\S+)
+
+Start
+  ^LLDP\s+neighbor-information\s+of\s+port\s+\d+\[${LOCAL_IF}\]
+  ^LLDP agent nearest-bridge:
```
Comment (FragmentedPacket) [inline templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm]:
```suggestion
  ^LLDP\s+agent\s+nearest-bridge:
  ^\s+LLDP\s+neighbor\s+index\s+:
```

Comment (xdai555) [reply templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm]: Fix it.

File: templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm (hunk 1 above)
Comment (xdai555) [reply templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm]: Yes,I have modified it and added some other changes based on cisco_ios_show_lldp_neighbors_detail.textfsm.

File: templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm (hunk 2 above)
Comment (xdai555) [reply templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm]: Fixed.

Comment (xdai555) [reply templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm]: It's fixed.

**Pull request merged**